Biotech/DefInjected/ThingDef/RangedMechanoid_Medium.xml

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<LanguageData>
+  
+  <!-- EN: toxic needle -->
+  <Bullet_ToxicNeedleGun.label>ядовитая игла</Bullet_ToxicNeedleGun.label>
+  
+  <!-- EN: beam graser -->
+  <Gun_BeamGraser.label>энергоплеть</Gun_BeamGraser.label>
+  <!-- EN: A high-energy gamma ray laser which fires a sweeping beam that burns and ignites targets. Because of the health risks, grasers are usually only used by mechanoids, and this one was made to interface directly with a mechanoid wielder and draw from its power source. -->
+  <Gun_BeamGraser.description>Мощный гамма-лазер, поджигающий цели своим лучом. Из-за потенциальных рисков для здоровья стрелка, обычно используется только механоидами. Конкретно этот образец был изготовлен специально для полной совместимости с механоидами и подключается непосредственно к их источникам питания.</Gun_BeamGraser.description>
+  <!-- EN: barrel -->
+  <Gun_BeamGraser.tools.barrel.label>ствол</Gun_BeamGraser.tools.barrel.label>
+  <!-- EN: beam graser -->
+  <Gun_BeamGraser.verbs.Verb_ShootBeam.label>энергоплеть</Gun_BeamGraser.verbs.Verb_ShootBeam.label>
+  
+  <!-- EN: toxic needle gun -->
+  <Gun_ToxicNeedle.label>ядовитый игломёт</Gun_ToxicNeedle.label>
+  <!-- EN: A mechanoid-built gun that fires toxin-infused metal needles. It can fire long distances and will poison the target in addition to doing physical damage. -->
+  <Gun_ToxicNeedle.description>Дальнобойное оружие механоидов, стреляющее металлическими иглами, покрытыми ядом. Иглы при попадании наносят физический урон и отравляют цель.  </Gun_ToxicNeedle.description>
+  <!-- EN: barrel -->
+  <Gun_ToxicNeedle.tools.barrel.label>ствол</Gun_ToxicNeedle.tools.barrel.label>
+  <!-- EN: toxic needle gun -->
+  <Gun_ToxicNeedle.verbs.Verb_Shoot.label>>ядовитый игломёт</Gun_ToxicNeedle.verbs.Verb_Shoot.label>
```

Comment:
```suggestion
  <Gun_ToxicNeedle.label>токсичный игломёт</Gun_ToxicNeedle.label>
  <!-- EN: A mechanoid-built gun that fires toxin-infused metal needles. It can fire long distances and will poison the target in addition to doing physical damage. -->
  <Gun_ToxicNeedle.description>Дальнобойное оружие механоидов, стреляющее металлическими иглами, покрытыми ядом. Иглы при попадании наносят физический урон и отравляют цель.  </Gun_ToxicNeedle.description>
  <!-- EN: barrel -->
  <Gun_ToxicNeedle.tools.barrel.label>ствол</Gun_ToxicNeedle.tools.barrel.label>
  <!-- EN: toxic needle gun -->
  <Gun_ToxicNeedle.verbs.Verb_Shoot.label>>токсичный игломёт</Gun_ToxicNeedle.verbs.Verb_Shoot.label>
```